assets/js/base/components/chip/style.scss

```diff
@@ -51,6 +51,20 @@
 	}
 }
 
+.theme-twentytwentyone {
+	// Adjust removable chip background color.
+	// @see https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/4404
```

Comment:
We don't usually add links to the issues in the code. In git, you can 'blame' a line or a file and find when it was last modified. This way, you can find the PR where it was introduced, which usually has a link to the issue. This allows tracking down why or when something was introduced without the need of manually adding comments in the code.

For example, this is the git blame version of this file:

https://github.com/woocommerce/woocommerce-gutenberg-products-block/blame/trunk/assets/js/base/components/chip/style.scss

Reply:
Thanks for pointing this out. I just updated this PR accordingly.